filebeat/module/apache2/ssl_request/_meta/fields.yml

```diff
@@ -0,0 +1,63 @@
+- name: ssl_access
```

Comment:
This creates an `ssl_access` namespace, but in the processor you use most fields form `apache.access`. It seems there is a big overlap with `apache2.access` so I quite like to put the fields there which are the same / identical if it makes sense. But the fields which are used from `access` don't have to be defined here also.

Reply:
Also this is in the `ssl_request` directory. There is some naming mess here, probably you decided to rename it during development :-)